lib/hsa/hsaTrackingUpdatePhase.hpp

```diff
@@ -141,16 +141,16 @@ class hsaPulsarUpdatePhase : public hsaCommand {
     uint second_now;
     uint second_last;
 
-    /// mutex lock prevent psr_coord to be read while it is being updated.
-    std::mutex _pulsar_lock;
+    /// mutex lock prevent beam_coord to be read while it is being updated.
+    std::mutex _beam_lock;
 
     /// Flag to avoid re-calculating freq-specific params except at first pass
     bool first_pass;
     /// Flag to trigger phase update, either because of endpt message or every second
     bool update_phase;
 
-    /// Endpoint for updating psr coordinates
-    std::string endpoint_psrcoord;
+    /// Endpoint for updating tracking coordinates
+    std::string endpoint_trackingcoord;
```

Comment:
```suggestion
    std::string endpoint_beam_coord;
```
This is maybe more consistent with the other names above.